news/_posts/2020-06-18-about-bc-breaks.md

```diff
@@ -0,0 +1,398 @@
+---
+layout: post
+title: "A deeper look at Backward Compatibility Breaks"
+subtitle: "What exactly is a BC break and how do they relate to PrestaShop"
+date:   2020-06-18
+authors: [ MathieuFerment ]
+image: /assets/images/theme/meta-logo-build.png
+icon: icon-code
+tags:
+---
+
+## A deeper look at Backward Compatibility Breaks
+
+This article explains what exactly are [Backward Compatibility](https://en.wikipedia.org/wiki/Backward_compatibility) Breaks also known as Breaking Changes or BC breaks and how it influences how an open source project like PrestaShop is built. It explains the different types of BC breaks, why they are needed sometimes and how PrestaShop maintainers manage them.
```

Comment:
```suggestion
This article explains what exactly are [Backward Compatibility](https://en.wikipedia.org/wiki/Backward_compatibility) Breaks (also known as Breaking Changes or BC) breaks and how it influences the development of an open source project like PrestaShop. It explains the different types of BC breaks, why they are sometimes needed, and how they are managed by PrestaShop maintainers.
```

> also known as Breaking Changes or BC

Did you realize we often use the same acronym differently?

- BC = Breaking change
- BC Break = Backwards Compatibility Break

I think this should be explained